src/ucp/core/ucp_ep.c

```diff
@@ -249,23 +260,46 @@ ucp_ep_local_disconnect_progress_remove_filter(const ucs_callbackq_elem_t *elem,
     return 1;
 }
 
-static UCS_F_NOINLINE void ucp_ep_remove_progress_callbacks(ucp_ep_h ep)
+static unsigned ucp_ep_set_failed_progress(void *arg)
 {
-    ucp_worker_h worker = ep->worker;
+    ucp_ep_set_failed_arg_t *set_ep_failed_arg = arg;
+    ucp_worker_h worker                        =
+            set_ep_failed_arg->ucp_ep->worker;
```

Comment:
minor: imo, better no alignment that this way

Reply:
fixed